pom.xml

```diff
@@ -99,6 +99,11 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.2.3</version>
+      </plugin>
```

Comment:
You can override the version of the surefire plugin by setting the `version.surefire.plugin` property rather than configuring the plugin in a whole XML block.